elements/media-quote/src/media-quote.js

```diff
@@ -23,6 +24,154 @@ class MediaQuote extends LitElement {
+            font-style: italic;
+            font-size: 0; /* Prevents a space between author and author detail comma on both sides  */
+            box-decoration-break: clone;
+          }
+            .author {
+              font-weight: var(--ddd-font-weight-bold);
+            }
+            .author, .author-detail {
+              font-size: var(--ddd-font-size-xxs);
+            }
+
+          figure {
+            width: 60%;
+            z-index: 0;
+          }
+
+          img {
+            width: 100%;
+          }
+
+          figcaption {
+            color: var(--ddd-theme-default-potentialMidnight); // TODO needs to contrast background color in dark mode
+            width: 100%;
+            border-top: var(--ddd-border-lg);
+            border-color: var(--ddd-theme-primary, var(--ddd-primary-1));
+            margin-top: var(--ddd-spacing-1);
+            padding-top: var(--ddd-spacing-1);
+          }
+
+          @container media-quote (max-width: 1261px) and (min-width: 1000px) {
+            .quote {
+              font-size: var(--ddd-font-size-xs);
+            }
+
+            .author, .author-detail {
+              font-size: var(--ddd-font-size-xxs);
+            }
+          }
+
+          @container media-quote (max-width: 999px) {
+            .quote {
+              position: relative;
+              width: 90%;
+              top: 0%;
+              left: 0%;
+              font-size: var(--ddd-font-size-xs);
+            }
+
+            .text-overlay {
+              text-align: center;
+            }
+
+            .author, .author-detail {
+              font-size: var(--ddd-font-size-xxs);
+            }
+
+            figure {
+              width: 100%;
+            }
+          }
+      `,];
+  }
+
+  render() {
+    return html`
+        <div class="media-quote-container">
+          <figure>
+            <div class="text-overlay"> 
+              <p class="quote">
+                <span class="content"><slot name="quote"></slot></span>
+                  ${this.querySelector('[slot="author"]') && this.querySelector('[slot="author"]').textContent.trim().length > 0 ? html`
+                    <span class="citation">
+                      <span class="author">- <slot name="author"></slot></span> 
+                      ${this.querySelector('[slot="author-detail"]') && this.querySelector('[slot="author-detail"]').textContent.trim().length > 0 ? html`
+                        <span class="author-detail">, <slot name="author-detail"></slot></span>
+                      ` : ''}
+                    </span>
+                  ` : ''}
+              </p>  
+            </div>
+            <img src="${this.src}" alt="${this.alt}">
+            ${this.querySelector('[slot="caption"]') && this.querySelector('[slot="caption"]').textContent.trim().length > 0 ? html`<figcaption><slot name="caption"></slot></figcaption>` : ''}
+          </figure>
+        </div>
+    `
+  }
+
+  static get properties() {
```

Comment:
Known bugs relating to caption opening and closing (will start fixing these on Monday):
* Triangle does not line up with the "Show Caption" or "Hide Caption" text
* When opening the caption, the quote text will shift down
* The caption text will sometimes overlap the border. May need to just set a width.

Not a bug but will also try to animate this opening and closing so it looks good rather than being sudden.